omps-integration.Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM fedora:latest
+
+RUN dnf -y install \
+    git \
+    gcc \
+    redhat-rpm-config \
+    popt-devel \
+    rpm-devel \
+    krb5-devel \
+    python3-devel \
+    python3-gunicorn \
+    python3-flask \
+    python3-jsonschema \
+    python3-koji \
+    python3-pyyaml \
+    python3-requests \
+    python3-tox \
+    && dnf -y clean all \
+    && rm -rf /tmp/*
+
+COPY . /operator-courier
+WORKDIR /operator-courier
+RUN pip3 install .
+
+# Install omps with this version of operator-courier.
+WORKDIR /omps
+RUN git clone https://github.com/release-engineering/operators-manifests-push-service.git .
```

Comment:
We can figure out how to version OMPS in the future if we want to test specific releases.